applications/tari_merge_mining_proxy/README.md

```diff
@@ -9,10 +9,14 @@ These instructions will be updated to include the development environment setup.
 ### Tari Base Node
 First we start tor by running the `start_tor.sh` script.
 
-Then we need to run the Tari Base Node with `cargo run --bin tari_base_node -- --create_id` and then 
-`cargo run --bin tari_base_node` 
+Then we need to run the Tari Base Node. If it is the first run then use `cargo run --bin tari_base_node -- --create_id` followed by 
+`cargo run --bin tari_base_node`. Otherwise just use `cargo run --bin tari_base_node`.
 
-Take note of the `grpc_address` in the config.toml file located in the `.tari` folder
+Take note of `proxy_host_address` in the config.toml file located in the `.tari` folder.
```

Comment:
This is a little misleading. We configure the `config.toml` file, but "taking note" suggests that someone/something is going to provide a value for us.

You could also easily have a config file that doesn't even have the `merge_mining_proxy section at all.

So I think this line needs to expand quite a bit:

* How to add / edit the `merge_mining_proxy` section
* How to find the right values for proxy host address if it isn't configured -- or if it's a matter of just choosing a port, then say so?

Reply:
Defaults should be provided by default.

Will add it though.

Reply:
Have expanded on this.